src/Driver.java

```diff
@@ -7,7 +9,78 @@ public static void main(String[] args) {
 		// TODO Write Code
 		System.out.println("Running...");
 		System.out.println("Done.");
+		UserInterface();
+
 		return;
 	}
 
+	public static void UserInterface() {
+
+		Scanner scanner = new Scanner(System.in);
+		System.out.println("Welcome to small co-op blah blah"); //fix introduction
+		String menu = "Make a selection, enter: \n"
+				+ "1) Enroll a member\n2) Remove a member\n3) Add a product\n"
+				+ "4) Check out a member's items\n5) Process a shipment\n"
+				+ "6) Change the price of a product\n7) Retrieve product info"
+				+ "\n8) Retrieve member info\n9) Print transactions\n"
+				+ "10) List all outstanding orders\n11) List all members with member info\n"
+				+ "12) List all products with product info\n13) Save\n14) Help \n"
+				+ "Press 0 at any time to quit the application";
+		System.out.println(menu);
+		boolean continueApplication = true;
+		while( continueApplication ) {
+			int userInput = scanner.nextInt();
```

Comment:
Add a prompt for the user.

`System.out.print("Enter a command:")`

Using `print()` instead of `println()` will look cleaner.

Reply:
sounds good I'll straighten it out.